### DISCUSSION TITLE
Where is v3 documentation?

### DISCUSSION THREAD

**Discussion by Lynxtickler** [serverless/serverless#12716] [Q&A]
I cannot find v3 documentation anywhere but would need to check a thing or two. Any idea where that can be found now that the official page is showing v4 only?

**Comment by resparzasoto** [comment]
@Lynxtickler You can find in the history of repository by tag version.

For example you can find the documentation of the page [Serverless.yml Reference](https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml#serverlessyml-reference) of the `serverless@3.39.0` in [serverless.yml.md](https://github.com/serverless/serverless/blob/v3.39.0/docs/providers/aws/guide/serverless.yml.md), if you need the documentation of another version, only replace the tag version in the URL.

I hope you find it useful!

**Comment by deanc** [comment]
Many folks are stuck on v3 due to the licensing change. It would be good if the FULL docs were published online somewhere still - not just the yml reference.

**Comment by privatejava** [comment]
Ok I could not bear anymore so I have created new repo just for docs. Here is the link for v3 docs using docs from v3.40.0. (Vibe coding for good cause)

https://privatejava.github.io/serverless-v3-docs/providers/aws/guide/serverless.yml/

Repository: https://github.com/privatejava/serverless-v3-docs (if you want to contribute)
